Makefile

```diff
@@ -95,6 +95,11 @@ build/webworker_dev.js: src/webworker.js
 	cp $< $@
 	sed -i -e 's#{{ PYODIDE_BASE_URL }}#./#g' $@
 
+update_base_url: \
+	build/console.html \
+	build/pyodide.js \
+	build/webworker.js \
+	docs/_build/html/console.html
```

Comment:
We should remove the last line. 

Currently S3 deployment fails with,
```
cp: cannot create regular file 'docs/_build/html/console.html': No such file or directory
make: *** [Makefile:83: docs/_build/html/console.html] Error 1
```

Reply:
Maybe add a `mkdir -p`?

Reply:
Makes sense. Will open a PR.

Reply:
Actually I really don't want to think too hard about when this needs or doesn't need to be updated so I'm gonna `mkdir -p`

Reply:
I can do the PR. I think the console.html deployed via docs for releases, might also not be correct with the current setup.

Reply:
Okay, I'll close mine then.

Reply:
Or you can take it over sure.